src/options.cpp

```diff
@@ -1917,8 +1917,8 @@ void options_manager::add_options_world_default()
     mOptionsSort["world_default"]++;
 
     add( "ZLEVELS", "world_default", translate_marker( "Experimental z-levels" ),
-         translate_marker( "If true, experimental z-level maps will be enabled.  This feature is not finished yet and turning it on will only slow the game down." ),
-         false
+         translate_marker( "If true, experimental z-level maps will be enabled. Turn this off if you experience excessive lag." ),
```

Comment:
```suggestion
         translate_marker( "If true, experimental z-level maps will be enabled. Turn this off if you experience excessive slowdown." ),
```
Lag is a more misnomer, whereas slowdown is a bit more professionally accepted, I believe.

Reply:
Sure, works for me